File: app/src/main/java/com/zh/userdemo/MainActivity.java
package com.zh.userdemo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.xtv_text).setOnClickListener(v->{
            startActivity(new Intent(MainActivity.this,XTextViewActivity.class));
        });

        findViewById(R.id.xtv_banner).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,XBannerViewActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/zh/userdemo/MainActivity.kt
package com.zh.userdemo

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        xtv_text.setOnClickListener { v: View? -> startActivity(Intent(this@MainActivity, XTextViewActivity::class.java)) }

        xtv_banner.setOnClickListener {
           startActivity(Intent(this@MainActivity,GlideDemoActivity::class.java))
        }
    }


}


